index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PraisyDram 2022</title>
+  <link
+    rel="stylesheet"
+    href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+  />
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <main>
+    <header class="header">
+      <i class="menu-icon fas fa-bars fa-2x"></i>
+      <a href="./index.html"><img src="./public/assets/icons/pd-logo.png" alt="PraisyDram Logo"></a>
```

Comment:
- [ ] You did a good job on the project but there is a little issue, You are supposed to stick to the Behance template as much as possible. Your design is missing the top sub-menu from the template. Please make sure your webpage has that top sub-menu and it matches the design on the template.

See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157688827-163fd13c-b7c7-4fe3-b7ed-a6094a501826.png)